test/unit/pipeline/test_predict_sklearn.py

```diff
@@ -83,6 +83,19 @@ def test_pre_process_input_data(
         for mock_func in [nonzero_flatliner_mock, replace_invalid_data_mock]:
             self.assertEqual(mock_func.call_count, 1)
 
+    # TODO include the test below to see if the pipeline uses the fallback forecast correctly
+    # This test requires a functioning 'happy_flow_pipeline' test which is not yet developed
+    # @patch("openstf.validation.validation.is_data_sufficient")
+    # def test_incomplete_input(self, is_data_sufficient_mock):
+    #     """Test if a fallback forecast is used when input is incomplete"""
+    #     input_data = TestData.load('input_data.pickle')
```

Comment:
Do we want this commented out code to be included in the code? Maybe it's better make an issue for the pipeline unittests and add this code as a suggestion to that issue instead of here.

Reply:
Good point. I think the code would be a logical place for this unittest-in-progress, but commented code isn't very nice. I've uncommented the code and added a @unittest.skip("reason for skipping").